doc/source/conf.py

```diff
@@ -63,6 +63,19 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath("../../python/"))
 
+import ray
+
+
+# Avoid @ray.remote run when doc generating
+def fake_remote(*args, **kwargs):
```

Comment:
quick question, probably too late, why do we need this here? I think ray.remote doesn't require a running ray cluster.